pkg/config/default_windows.go

```diff
@@ -49,3 +49,8 @@ func getLibpodTmpDir() string {
 func getDefaultMachineVolumes() []string {
 	return []string{}
 }
+
+func getDefaultComposeProviders() []string {
+	// Rely on os.LookPath to do the trick on Windows.
+	return []string{"docker-compose", "podman-compose"}
```

Comment:
did you check where the docker-compose plugin is installed on windows? cc @benoitf